packages/trader/src/Modules/Trading/Containers/trade-chart.tsx

```diff
@@ -104,7 +104,7 @@ const TradeChart = observer((props: TTradeChartProps) => {
             crosshairTooltipLeftAllow={560}
             showLastDigitStats={isDesktop() ? show_digits_stats : false}
             chartControlsWidgets={null}
-            chartStatusListener={(v: boolean) => setChartStatus(!v)}
+            chartStatusListener={(v: boolean) => setChartStatus(!v, true)}
```

Comment:
we need to make sure that this is always called after `setChartStatus(true)` in `onMount()`, hence passing an extra argument in order to debounce the call inside `setChartStatus`.